src/formatstringplus/widget/formatstringplus.js

```diff
@@ -28,7 +37,7 @@ dojo.declare('formatstringplus.widget.formatstringplus', mxui.widget._WidgetBase
     },
 
     update : function(obj, callback){
-        dojo.empty(this.domNode);
+        construct.empty(this.domNode);
         

```

Comment:
Add `this.unsubscribeAll();` in the update function

Reply:
Done